upgrade the binutils fixes the issue and VX z13 build error can be triggered :

```Bash
loops_comparison.dispatch.c.src: In function ‘simd_binary_equal_b8’:

numpy/core/src/common/simd/vec/operators.h:148:22: error: ‘__builtin_s390_vec_eqv’ requires z14 or higher

  148 | #define npyv_xnor_b8 vec_eqv

      |                      ^~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src:236:21: note: in expansion of macro ‘npyv_xnor_b8’

  236 |         npyv_b8 c = npyv_@VOP@_b8(a, b);

      |                     ^~~~~~~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src: In function ‘simd_binary_scalar1_equal_b8’:

numpy/core/src/common/simd/vec/operators.h:148:22: error: ‘__builtin_s390_vec_eqv’ requires z14 or higher

  148 | #define npyv_xnor_b8 vec_eqv

      |                      ^~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src:260:21: note: in expansion of macro ‘npyv_xnor_b8’

  260 |         npyv_b8 c = npyv_@VOP@_b8(a, b);

      |                     ^~~~~~~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src: In function ‘simd_binary_scalar2_equal_b8’:

numpy/core/src/common/simd/vec/operators.h:148:22: error: ‘__builtin_s390_vec_eqv’ requires z14 or higher

  148 | #define npyv_xnor_b8 vec_eqv

      |                      ^~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src:283:21: note: in expansion of macro ‘npyv_xnor_b8’

  283 |         npyv_b8 c = npyv_@VOP@_b8(a, b);

      |                     ^~~~~~~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src: In function ‘simd_binary_less_equal_b8’:

numpy/core/src/common/simd/vec/operators.h:147:21: error: ‘__builtin_s390_vec_orc’ requires z14 or higher

  147 | #define npyv_orc_b8 vec_orc

      |                     ^~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src:236:21: note: in expansion of macro ‘npyv_orc_b8’

  236 |         npyv_b8 c = npyv_@VOP@_b8(a, b);

      |                     ^~~~~~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src: In function ‘simd_binary_scalar1_less_equal_b8’:

numpy/core/src/common/simd/vec/operators.h:147:21: error: ‘__builtin_s390_vec_orc’ requires z14 or higher

  147 | #define npyv_orc_b8 vec_orc

      |                     ^~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src:260:21: note: in expansion of macro ‘npyv_orc_b8’

  260 |         npyv_b8 c = npyv_@VOP@_b8(a, b);

      |                     ^~~~~~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src: In function ‘simd_binary_scalar2_less_equal_b8’:

numpy/core/src/common/simd/vec/operators.h:147:21: error: ‘__builtin_s390_vec_orc’ requires z14 or higher

  147 | #define npyv_orc_b8 vec_orc

      |                     ^~~~~~~

numpy/core/src/umath/loops_comparison.dispatch.c.src:283:21: note: in expansion of macro ‘npyv_orc_b8’

  283 |         npyv_b8 c = npyv_@VOP@_b8(a, b);
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -38,6 +38,9 @@ jobs:
     - python: "3.8"
       os: linux
       arch: s390x
+      # fixes VX assembler ambiguous errors
+      # due to compiler incompatibility
+      install: sudo apt update && sudo apt -y --only-upgrade install binutils
       env:
        # use OpenBLAS build, not system ATLAS
        - DOWNLOAD_OPENBLAS=1

